docs/Anjay_integrations/Anjay_Nordic_integration/nRF9160.md

```diff
@@ -67,7 +74,7 @@ To connect the board:
     3. In the **Device credentials** step:
          - In the **Device ID** enter your board endpoint name, e.g. `test_device`.
              ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
-         - In the **Security mode** section, select the **PSK** mode:
+         - In the **Security mode** section, select the **PSK (Pre-Shared Key)** mode:
               - In the **Key identity** field, type `test_device`
```

Comment:
```suggestion
              - In the **Key identity** field, type the same name as in the `Endpoint name` field
```

Reply:
Or remove this section completely, because the Key identity field is filled automatically.